lib/osf-components/addon/components/tags-widget/template.hbs

```diff
@@ -22,6 +22,7 @@
         data-test-tags-widget-tag='{{tag}}'
         data-analytics-name='Click tag'
         role='link'
```

Comment:
`role='link'` seems inappropriate for the case when clicking this doesn't do anything, so we should wrap this in a conditional too

Reply:
We can likely remove the `class` stuff below if this is no longer `role='link'` as well